code/osx32_start.S

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2016-2017 Andrew Kallmeyer <fsmv@sapium.net>
+// Provided under the MIT License: https://mit-license.org
+
+// Thanks to Franc[e]sco (weeb.ddns.net) for his nolibc projects which I used
```

Comment:
Lets just leave this thanks message in the linux one only.

This one isn't as closely derived from his stuff and I need to update his website anyway.

Reply:
Updated in 8e80d67